[BUG] Error building with lely-canopen

### Description / Steps to reproduce the issue

Recently tried to build and image with lely-canopen turned on and got tons of errors. Tried different versions of lely-canopen, but without any success. So question is - does it work or abandoned?

My build environment is ESP32-WLROOM32 MCU. 

Error is the following:

```sh
make
Create version.h
Downloading: lely-core-637de61625c9e8af2b9598109d6fb0e3a2829ce6.tar.gz
  % Total    % Received % Xferd  Average Speed  Time    Time    Time   Current
                                 Dload  Upload  Total   Spent   Left   Speed
  0      0   0      0   0      0      0      0                              0
100 810.5k   0 810.5k   0      0 694.7k      0           00:01              0
Unpacking: lely-core-637de61625c9e8af2b9598109d6fb0e3a2829ce6.tar.gz -> lely-core-master-637de61625c9e8af2b9598109d6fb0e3a2829ce6
Patching lely-core
LN: platform/board to <path>/nuttxspace/apps/platform/dummy
Register: dd
Register: nsh
Register: sh
Espressif HAL for 3rd Party Platforms: initializing submodules...
Applying patches...
CC:  <path>/nuttxspace/apps/canutils/lely-canopen/lelyCC:  <path>/nuttxspace/apps/canutils/lely-canopen/lelyCC:  <path>/nuttxspace/apps/canutils/lely-canopen/lelyCC:  <path>/nuttxspace/apps/canutils/lely-canopen/lelyCC:  <path>/nuttxspace/apps/canutils/lely-canopen/lelyCC:  <path>/nuttxspace/apps/canutils/lely-canopen/lelyCC:  <path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/dev.c In file included from <path>/nuttxspace/apps/canutils/lely-canopen/lely-core/include/lely/co/detail/obj.h:27,
                 from <path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/dev.c:26:
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/include/lely/co/val.h:787:66: warning: 'struct __frbuf' declared inside parameter list will not be visible outside of this definition or declaration
  787 | size_t co_val_read_frbuf(co_unsigned16_t type, void *val, struct __frbuf *buf);
      |                                                                  ^~~~~~~
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/include/lely/co/val.h:869:63: warning: 'struct __fwbuf' declared inside parameter list will not be visible outside of this definition or declaration
  869 |                 co_unsigned16_t type, const void *val, struct __fwbuf *buf);
      |                                                               ^~~~~~~
CC:  <path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c In file included from <path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c:44:
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/include/lely/co/val.h:787:66: warning: 'struct __frbuf' declared inside parameter list will not be visible outside of this definition or declaration
  787 | size_t co_val_read_frbuf(co_unsigned16_t type, void *val, struct __frbuf *buf);
      |                                                                  ^~~~~~~
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/include/lely/co/val.h:869:63: warning: 'struct __fwbuf' declared inside parameter list will not be visible outside of this definition or declaration
  869 |                 co_unsigned16_t type, const void *val, struct __fwbuf *buf);
      |                                                               ^~~~~~~
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c: In function 'co_nmt_cfg_ind':
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c:2394:18: error: 'co_nmt_t' {aka 'struct __co_nmt'} has no member named 'cfg_ind'; did you mean 'cs_ind'?
 2394 |         if (nmt->cfg_ind) {
      |                  ^~~~~~~
      |                  cs_ind
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c:2395:22: error: 'co_nmt_t' {aka 'struct __co_nmt'} has no member named 'cfg_ind'; did you mean 'cs_ind'?
 2395 |                 nmt->cfg_ind(nmt, id, sdo, nmt->cfg_data);
      |                      ^~~~~~~
      |                      cs_ind
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c:2395:49: error: 'co_nmt_t' {aka 'struct __co_nmt'} has no member named 'cfg_data'; did you mean 'cs_data'?
 2395 |                 nmt->cfg_ind(nmt, id, sdo, nmt->cfg_data);
      |                                                 ^~~~~~~~
      |                                                 cs_data
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c: In function 'co_nmt_cfg_con':
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c:2408:42: error: 'co_nmt_t' {aka 'struct __co_nmt'} has no member named 'slaves'
 2408 |         struct co_nmt_slave *slave = &nmt->slaves[id - 1];
      |                                          ^~
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c:2409:14: error: invalid use of undefined type 'struct co_nmt_slave'
 2409 |         slave->configuring = 0;
      |              ^~
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c:2410:33: error: invalid use of undefined type 'struct co_nmt_slave'
 2410 |         co_nmt_cfg_destroy(slave->cfg);
      |                                 ^~
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c:2411:14: error: invalid use of undefined type 'struct co_nmt_slave'
 2411 |         slave->cfg = NULL;
      |              ^~
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c:2418:35: error: implicit declaration of function 'co_nmt_hb_find'; did you mean 'co_nmt_hb_fini'? [-Wimplicit-function-declaration]
 2418 |                 co_nmt_hb_t *hb = co_nmt_hb_find(nmt, id, &ms);
      |                                   ^~~~~~~~~~~~~~
      |                                   co_nmt_hb_fini
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c:2418:35: error: initialization of 'co_nmt_hb_t *' {aka 'struct __co_nmt_hb *'} from 'int' makes pointer from integer without a cast [-Wint-conversion]
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c:2426:18: error: invalid use of undefined type 'struct co_nmt_slave'
 2426 |         if (slave->cfg_con)
      |                  ^~
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c:2427:22: error: invalid use of undefined type 'struct co_nmt_slave'
 2427 |                 slave->cfg_con(nmt, id, ac, slave->cfg_data);
      |                      ^~
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c:2427:50: error: invalid use of undefined type 'struct co_nmt_slave'
 2427 |                 slave->cfg_con(nmt, id, ac, slave->cfg_data);
      |                                                  ^~
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c: In function 'co_1016_dn_ind':
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c:2573:24: error: 'co_nmt_t' {aka 'struct __co_nmt'} has no member named 'slaves'
 2573 |                 if (nmt->slaves[id - 1].cfg)
      |                        ^~
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c: In function 'co_1f25_dn_ind':
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c:2645:40: error: 'co_nmt_t' {aka 'struct __co_nmt'} has no member named 'slaves'
 2645 |                         || (id && !(nmt->slaves[id - 1].assignment & 0x01)))
      |                                        ^~
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c:2649:17: error: 'co_nmt_t' {aka 'struct __co_nmt'} has no member named 'master'
 2649 |         if (!nmt->master || val.u32 != UINT32_C(0x666e6f63))
      |                 ^~
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c:2664:24: error: 'co_nmt_t' {aka 'struct __co_nmt'} has no member named 'slaves'
 2664 |                 if (nmt->slaves[id - 1].cfg)
      |                        ^~
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c:2666:44: error: 'co_nmt_t' {aka 'struct __co_nmt'} has no member named 'timeout'
 2666 |                 co_nmt_cfg_req(nmt, id, nmt->timeout, NULL, NULL);
      |                                            ^~
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c:2678:58: error: 'co_nmt_t' {aka 'struct __co_nmt'} has no member named 'slaves'
 2678 |                         struct co_nmt_slave *slave = &nmt->slaves[id - 1];
      |                                                          ^~
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c:2681:36: error: invalid use of undefined type 'struct co_nmt_slave'
 2681 |                         if (!(slave->assignment & 0x01) || slave->configuring)
      |                                    ^~
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c:2681:65: error: invalid use of undefined type 'struct co_nmt_slave'
 2681 |                         if (!(slave->assignment & 0x01) || slave->configuring)
      |                                                                 ^~
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c:2683:52: error: 'co_nmt_t' {aka 'struct __co_nmt'} has no member named 'timeout'
 2683 |                         co_nmt_cfg_req(nmt, id, nmt->timeout, NULL, NULL);
      |                                                    ^~
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c: In function 'co_nmt_dn_ind':
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c:3134:18: error: 'co_nmt_t' {aka 'struct __co_nmt'} has no member named 'dn_ind'; did you mean 'cs_ind'?
 3134 |         if (nmt->dn_ind)
      |                  ^~~~~~
      |                  cs_ind
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c:3135:22: error: 'co_nmt_t' {aka 'struct __co_nmt'} has no member named 'dn_ind'; did you mean 'cs_ind'?
 3135 |                 nmt->dn_ind(nmt, co_csdo_get_num(sdo), idx, subidx, size, nbyte,
      |                      ^~~~~~
      |                      cs_ind
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c:3136:38: error: 'co_nmt_t' {aka 'struct __co_nmt'} has no member named 'dn_data'; did you mean 'cs_data'?
 3136 |                                 nmt->dn_data);
      |                                      ^~~~~~~
      |                                      cs_data
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c: In function 'co_nmt_up_ind':
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c:3146:18: error: 'co_nmt_t' {aka 'struct __co_nmt'} has no member named 'up_ind'; did you mean 'cs_ind'?
 3146 |         if (nmt->up_ind)
      |                  ^~~~~~
      |                  cs_ind
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c:3147:22: error: 'co_nmt_t' {aka 'struct __co_nmt'} has no member named 'up_ind'; did you mean 'cs_ind'?
 3147 |                 nmt->up_ind(nmt, co_csdo_get_num(sdo), idx, subidx, size, nbyte,
      |                      ^~~~~~
      |                      cs_ind
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c:3148:38: error: 'co_nmt_t' {aka 'struct __co_nmt'} has no member named 'up_data'; did you mean 'cs_data'?
 3148 |                                 nmt->up_data);
      |                                      ^~~~~~~
      |                                      cs_data
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c: At top level:
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c:3140:1: warning: 'co_nmt_up_ind' defined but not used [-Wunused-function]
 3140 | co_nmt_up_ind(const co_csdo_t *sdo, co_unsigned16_t idx, co_unsigned8_t subidx,
      | ^~~~~~~~~~~~~
<path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c:3128:1: warning: 'co_nmt_dn_ind' defined but not used [-Wunused-function]
 3128 | co_nmt_dn_ind(const co_csdo_t *sdo, co_unsigned16_t idx, co_unsigned8_t subidx,
      | ^~~~~~~~~~~~~
make[2]: *** [<path>/nuttxspace/apps/Application.mk:239: <path>/nuttxspace/apps/canutils/lely-canopen/lely-core/src/co/nmt.c.home.roman.Projects.Завод.mcu.nuttx-zavod.nuttxspace.apps.canutils.lely-canopen.o] Error 1
make[1]: *** [Makefile:54: <path>/nuttxspace/apps/canutils/lely-canopen_all] Error 2
make: *** [tools/LibTargets.mk:248: <path>/nuttxspace/apps/libapps.a] Error 2

```

### On which OS does this issue occur?

[OS: Linux]

### What is the version of your OS?

6.12.93 #1-NixOS

### NuttX Version

12.13.0

### Issue Architecture

[Arch: xtensa]

### Issue Area

[Area: CAN]

### Host information

```sh

make host_info
file sysinfo.h not exists
NuttX configuration options:
  CONFIG_HOST_LINUX=y
  CONFIG_APPS_DIR="../apps"
  CONFIG_BASE_DEFCONFIG="esp32-devkitc:nsh-dirty"
  CONFIG_BUILD_FLAT=y
  CONFIG_INTELHEX_BINARY=y
  CONFIG_OUTPUT_STRIP_PATHS=y
  CONFIG_ARCH_HAVE_STDARG_H=y
  CONFIG_ARCH_HAVE_SETJMP=y
  CONFIG_ARCH_NONE_DEBUG_H=y
  CONFIG_FORTIFY_SOURCE=0
  CONFIG_NDEBUG=y
  CONFIG_DEBUG_ALERT=y
  CONFIG_ARCH_HAVE_STACKCHECK=y
  CONFIG_STACK_USAGE_WARNING=0
  CONFIG_ARCH_HAVE_CUSTOMOPT=y
  CONFIG_DEBUG_FULLOPT=y
  CONFIG_LTO_NONE=y
  CONFIG_DEBUG_OPT_UNUSED_SECTIONS=y
  CONFIG_DEBUG_LINK_MAP=y
  CONFIG_ARCH_XTENSA=y
  CONFIG_ARCH="xtensa"
  CONFIG_ARCH_CHIP="esp32"
  CONFIG_ESPRESSIF_CHIP_SERIES="esp32"
  CONFIG_ESPRESSIF_NUM_CPUS=2
  CONFIG_ESPRESSIF_SIMPLE_BOOT=y
  CONFIG_ESPRESSIF_FLASH_MODE_DIO=y
  CONFIG_ESPRESSIF_FLASH_FREQ_40M=y
  CONFIG_ESPRESSIF_FLASH_FREQ="40m"
  CONFIG_ARCH_CHIP_ESP32=y
  CONFIG_ARCH_FAMILY_LX6=y
  CONFIG_XTENSA_CP_INITSET=0x0001
  CONFIG_XTENSA_TOOLCHAIN_ESP=y
  CONFIG_ARCH_CHIP_ESP32WROOM32=y
  CONFIG_ESP32_DUAL_CPU=y
  CONFIG_ESP32_FLASH_4M=y
  CONFIG_ESP32_FLASH_DETECT=y
  CONFIG_ESP32_ESP32DXWDXX=y
  CONFIG_ESP32_DEFAULT_CPU_FREQ_240=y
  CONFIG_ESP32_DEFAULT_CPU_FREQ_MHZ=240
  CONFIG_ESP32_XTAL_40MZ=y
  CONFIG_ESP_PHY_IRQSTATE_FLAGS_NUMBER=3
  CONFIG_ESP32_UART=y
  CONFIG_ESP32_TWAI=y
  CONFIG_ESP32_RTC=y
  CONFIG_ESP32_UART0=y
  CONFIG_ESP32_TWAI0=y
  CONFIG_ESP32_BT_RESERVE_DRAM=0
  CONFIG_ESP32_TRACEMEM_RESERVE_DRAM=0
  CONFIG_ESP32_ULP_COPROC_RESERVE_MEM=0
  CONFIG_ESP32_UART0_TXPIN=1
  CONFIG_ESP32_UART0_RXPIN=3
  CONFIG_ESP32_TWAI0_TXPIN=21
  CONFIG_ESP32_TWAI0_RXPIN=22
  CONFIG_ESP32_TWAI0_BITRATE=1000000
  CONFIG_ESP32_TWAI0_SAMPLEP=80
  CONFIG_ESP32_TWAI0_SJW=3
  CONFIG_ESP32_FLASH_MODE_DIO=y
  CONFIG_ESP32_FLASH_FREQ_40M=y
  CONFIG_ESP32_RTC_CLK_SRC_INT_RC=y
  CONFIG_ARCH_TOOLCHAIN_GNU=y
  CONFIG_ARCH_VECNOTIRQ=y
  CONFIG_ARCH_HAVE_TEXT_HEAP=y
  CONFIG_ARCH_HAVE_MULTICPU=y
  CONFIG_ARCH_HAVE_FPU=y
  CONFIG_ARCH_HAVE_MPU=y
  CONFIG_ARCH_HAVE_RESET=y
  CONFIG_ARCH_HAVE_TESTSET=y
  CONFIG_ARCH_HAVE_THREAD_LOCAL=y
  CONFIG_ARCH_HAVE_SYSCALL_HOOKS=y
  CONFIG_ARCH_HAVE_BACKTRACE=y
  CONFIG_ARCH_HAVE_DEBUG=y
  CONFIG_ARCH_HAVE_PERF_EVENTS=y
  CONFIG_ARCH_HAVE_BOOTLOADER=y
  CONFIG_ARCH_HAVE_CPUINFO=y
  CONFIG_ARCH_CPUINFO_FREQ_KHZ=0
  CONFIG_ARCH_HAVE_TCBINFO=y
  CONFIG_ARCH_FPU=y
  CONFIG_NCPUS=1
  CONFIG_ARCH_STACKDUMP=y
  CONFIG_ARCH_STACKDUMP_MAX_LENGTH=0
  CONFIG_BOARD_LOOPSPERMSEC=16717
  CONFIG_ARCH_HAVE_INTERRUPTSTACK=y
  CONFIG_ARCH_INTERRUPTSTACK=0
  CONFIG_BOOT_RUNFROMFLASH=y
  CONFIG_RAM_START=0x20000000
  CONFIG_RAM_SIZE=114688
  CONFIG_FLASH_START=0x0
  CONFIG_FLASH_SIZE=0
  CONFIG_ARCH_BOARD_ESP32_DEVKITC=y
  CONFIG_ARCH_BOARD="esp32-devkitc"
  CONFIG_ARCH_HAVE_LEDS=y
  CONFIG_ARCH_HAVE_BUTTONS=y
  CONFIG_ARCH_BOARD_COMMON=y
  CONFIG_BOARD_CRASHDUMP_NONE=y
  CONFIG_BOARDCTL=y
  CONFIG_BOARDCTL_MKRD=y
  CONFIG_BOARD_MEMORY_RANGE=""
  CONFIG_DISABLE_OS_API=y
  CONFIG_USEC_PER_TICK=10000
  CONFIG_TIMER_ADJUST_USEC=0
  CONFIG_START_YEAR=2011
  CONFIG_START_MONTH=12
  CONFIG_START_DAY=6
  CONFIG_PREALLOC_TIMERS=4
  CONFIG_IRQ_NWORKS=8
  CONFIG_IRQ_WORK_SECTION=""
  CONFIG_IRQ_WORK_STACKSIZE=2048
  CONFIG_INIT_ENTRY=y
  CONFIG_INIT_ARGS=""
  CONFIG_INIT_STACKSIZE=3072
  CONFIG_INIT_PRIORITY=100
  CONFIG_INIT_ENTRYPOINT="nsh_main"
  CONFIG_INIT_ENTRYNAME="nsh_main"
  CONFIG_RR_INTERVAL=200
  CONFIG_TASK_NAME_SIZE=31
  CONFIG_SCHED_WAITPID=y
  CONFIG_PTHREAD_MUTEX_BOTH=y
  CONFIG_PTHREAD_MUTEX_DEFAULT_UNSAFE=y
  CONFIG_PTHREAD_MUTEX_DEFAULT_PRIO_NONE=y
  CONFIG_SCHED_CPULOAD_NONE=y
  CONFIG_SCHED_CPULOAD_TICKSPERSEC=100
  CONFIG_SCHED_PROFILE_TICKSPERSEC=1000
  CONFIG_SCHED_STACK_RECORD=0
  CONFIG_DEV_CONSOLE=y
  CONFIG_NFILE_DESCRIPTORS_PER_BLOCK=8
  CONFIG_FILE_STREAM=y
  CONFIG_ENABLE_ALL_SIGNALS=y
  CONFIG_SIG_PREALLOC_ACTIONS=4
  CONFIG_SIG_ALLOC_ACTIONS=1
  CONFIG_SIG_PREALLOC_IRQ_ACTIONS=8
  CONFIG_PREALLOC_MQ_MSGS=8
  CONFIG_PREALLOC_MQ_IRQ_MSGS=8
  CONFIG_MQ_MAXMSGSIZE=32
  CONFIG_DEFAULT_TASK_STACKSIZE=2048
  CONFIG_IDLETHREAD_STACKSIZE=3072
  CONFIG_PTHREAD_STACK_MIN=256
  CONFIG_PTHREAD_STACK_DEFAULT=2048
  CONFIG_PTHREAD_GUARDSIZE_DEFAULT=0
  CONFIG_PID_INITIAL_COUNT=16
  CONFIG_CAN=y
  CONFIG_CAN_TXFIFOSIZE=8
  CONFIG_CAN_RXFIFOSIZE=8
  CONFIG_CAN_NPENDINGRTR=4
  CONFIG_ARCH_HAVE_I2CRESET=y
  CONFIG_DEV_NULL=y
  CONFIG_DEV_ZERO=y
  CONFIG_DEV_OPTEE_NONE=y
  CONFIG_DRVR_MKRD=y
  CONFIG_NETDEVICES=y
  CONFIG_MDIO_BUS=y
  CONFIG_PIPES=y
  CONFIG_DEV_PIPE_MAXSIZE=65535
  CONFIG_DEV_PIPE_SIZE=1024
  CONFIG_DEV_FIFO_SIZE=1024
  CONFIG_DEV_PIPE_VFS_PATH="/var/pipe"
  CONFIG_DEV_PIPE_NPOLLWAITERS=4
  CONFIG_ARCH_HAVE_SERIAL_TERMIOS=y
  CONFIG_SERIAL=y
  CONFIG_SERIAL_CONSOLE=y
  CONFIG_MCU_SERIAL=y
  CONFIG_STANDARD_SERIAL=y
  CONFIG_SERIAL_NPOLLWAITERS=4
  CONFIG_UART0_SERIAL_CONSOLE=y
  CONFIG_UART0_SERIALDRIVER=y
  CONFIG_UART0_RXBUFSIZE=256
  CONFIG_UART0_TXBUFSIZE=256
  CONFIG_UART0_BAUD=115200
  CONFIG_UART0_BITS=8
  CONFIG_UART0_PARITY=0
  CONFIG_UART0_2STOP=0
  CONFIG_SYSLOG=y
  CONFIG_SYSLOG_DEFAULT_MASK=0xff
  CONFIG_SYSLOG_CRLF=y
  CONFIG_SYSLOG_MAX_CHANNELS=1
  CONFIG_SYSLOG_BUFFER=y
  CONFIG_SYSLOG_BUFSIZE=64
  CONFIG_SYSLOG_DEVPATH="/dev/ttyS1"
  CONFIG_SYSLOG_DEFAULT=y
  CONFIG_ARCH_HAVE_NET=y
  CONFIG_NET_READAHEAD=y
  CONFIG_NET=y
  CONFIG_NET_DEFAULT_MIN_PORT=4096
  CONFIG_NET_DEFAULT_MAX_PORT=32000
  CONFIG_NET_ETH_PKTSIZE=590
  CONFIG_NET_GUARDSIZE=2
  CONFIG_NET_LL_GUARDSIZE=14
  CONFIG_NET_RECV_BUFSIZE=0
  CONFIG_NET_ETHERNET=y
  CONFIG_NETDEV_IFINDEX=y
  CONFIG_NET_IPv4=y
  CONFIG_NET_IPFORWARD_ALLOC_STRUCT=1
  CONFIG_NET_PREALLOC_DEVIF_CALLBACKS=16
  CONFIG_NET_ALLOC_DEVIF_CALLBACKS=0
  CONFIG_NET_SOCKOPTS=y
  CONFIG_NET_CANPROTO_OPTIONS=y
  CONFIG_NET_IPV4_CHECKSUMS=y
  CONFIG_NET_CAN=y
  CONFIG_CAN_PREALLOC_CONNS=4
  CONFIG_CAN_ALLOC_CONNS=0
  CONFIG_NET_CAN_SOCK_OPTS=y
  CONFIG_NET_CAN_RAW_FILTER_MAX=32
  CONFIG_NET_CAN_NPOLLWAITERS=4
  CONFIG_NET_ICMP=y
  CONFIG_NET_ICMP_PMTU_ENTRIES=0
  CONFIG_NET_ICMP_CHECKSUMS=y
  CONFIG_NET_IGMP_CHECKSUMS=y
  CONFIG_NET_ARP=y
  CONFIG_NET_ARPTAB_SIZE=16
  CONFIG_NET_ARP_MAXAGE=120
  CONFIG_NET_ARP_MAXAGE_UNREACHABLE=1
  CONFIG_NET_ARP_SEND=y
  CONFIG_ARP_SEND_MAXTRIES=5
  CONFIG_ARP_SEND_DELAYMSEC=20
  CONFIG_NET_SNOOP_BUFSIZE=4096
  CONFIG_NET_RECV_PACK=y
  CONFIG_FS_NEPOLL_DESCRIPTORS=8
  CONFIG_FS_LOCK_BUCKET_SIZE=0
  CONFIG_SENDFILE_BUFSIZE=512
  CONFIG_FS_BACKTRACE=0
  CONFIG_FS_MQUEUE_VFS_PATH="/var/mqueue"
  CONFIG_FS_MQUEUE_NPOLLWAITERS=4
  CONFIG_FS_ANONMAP=y
  CONFIG_FS_PROCFS=y
  CONFIG_NXFONTS_PACKEDMSFIRST=y
  CONFIG_MM_DEFAULT_MANAGER=y
  CONFIG_MM_DEFAULT_ALIGNMENT=8
  CONFIG_MM_NODE_GUARDSIZE=0
  CONFIG_MM_REGIONS=3
  CONFIG_MM_MAP_COUNT_MAX=1024
  CONFIG_MM_BACKTRACE=-1
  CONFIG_MM_FREE_DELAYCOUNT_MAX=0
  CONFIG_MM_HEAP_BIGGEST_COUNT=30
  CONFIG_MM_HEAP_MEMPOOL_THRESHOLD=-1
  CONFIG_MM_IOB=y
  CONFIG_IOB_NBUFFERS=24
  CONFIG_IOB_BUFSIZE=196
  CONFIG_IOB_ALIGNMENT=4
  CONFIG_IOB_SECTION=""
  CONFIG_IOB_NCHAINS=24
  CONFIG_IOB_THROTTLE=0
  CONFIG_BINFMT_ELF_RELOCATABLE=y
  CONFIG_STDIO_BUFFER_SIZE=64
  CONFIG_STDIO_LINEBUFFER=y
  CONFIG_NUNGET_CHARS=2
  CONFIG_LIBC_FLOATINGPOINT=y
  CONFIG_LIBC_LONG_LONG=y
  CONFIG_ARCH_LOWPUTC=y
  CONFIG_LIBC_ARCH_MEMCHR=y
  CONFIG_LIBC_ARCH_MEMCPY=y
  CONFIG_LIBC_ARCH_MEMCMP=y
  CONFIG_LIBC_ARCH_MEMMOVE=y
  CONFIG_LIBC_ARCH_MEMSET=y
  CONFIG_LIBC_ARCH_STRCHR=y
  CONFIG_LIBC_ARCH_STRCMP=y
  CONFIG_LIBC_ARCH_STRCPY=y
  CONFIG_LIBC_ARCH_STRLCPY=y
  CONFIG_LIBC_ARCH_STRNCPY=y
  CONFIG_LIBC_ARCH_STRLEN=y
  CONFIG_LIBC_ARCH_STRNLEN=y
  CONFIG_LIBC_PREVENT_STRING_KERNEL=y
  CONFIG_LIBC_PREVENT_MEMCHR_KERNEL=y
  CONFIG_LIBC_PREVENT_MEMCMP_KERNEL=y
  CONFIG_LIBC_PREVENT_MEMCPY_KERNEL=y
  CONFIG_LIBC_PREVENT_MEMMOVE_KERNEL=y
  CONFIG_LIBC_PREVENT_MEMSET_KERNEL=y
  CONFIG_LIBC_PREVENT_STRCAT_KERNEL=y
  CONFIG_LIBC_PREVENT_STRCASECMP_KERNEL=y
  CONFIG_LIBC_PREVENT_STRCHR_KERNEL=y
  CONFIG_LIBC_PREVENT_STRCHRNUL_KERNEL=y
  CONFIG_LIBC_PREVENT_STRCMP_KERNEL=y
  CONFIG_LIBC_PREVENT_STRCPY_KERNEL=y
  CONFIG_LIBC_PREVENT_STRLCAT_KERNEL=y
  CONFIG_LIBC_PREVENT_STRLEN_KERNEL=y
  CONFIG_LIBC_PREVENT_STRLCPY_KERNEL=y
  CONFIG_LIBC_PREVENT_STRNCASECMP_KERNEL=y
  CONFIG_LIBC_PREVENT_STRNCAT_KERNEL=y
  CONFIG_LIBC_PREVENT_STRNLEN_KERNEL=y
  CONFIG_LIBC_PREVENT_STRNCMP_KERNEL=y
  CONFIG_LIBC_PREVENT_STRNCPY_KERNEL=y
  CONFIG_LIBC_PREVENT_STRRCHR_KERNEL=y
  CONFIG_LIBC_RAND_ORDER=1
  CONFIG_LIBC_HOMEDIR="/"
  CONFIG_LIBC_TMPDIR="/tmp"
  CONFIG_LIBC_MAX_TMPFILE=32
  CONFIG_LIBC_MAX_EXITFUNS=0
  CONFIG_POSIX_SPAWN_DEFAULT_STACKSIZE=2048
  CONFIG_LIBC_HOSTNAME=""
  CONFIG_LIBC_OPEN_MAX=256
  CONFIG_NAME_MAX=32
  CONFIG_PATH_MAX=256
  CONFIG_LINE_MAX=64
  CONFIG_LIBC_STRERROR_ERRNUM=y
  CONFIG_LIBC_STRSIGNAL=y
  CONFIG_TLS_NELEM=0
  CONFIG_TLS_TASK_NELEM=0
  CONFIG_TLS_NCLEANUP=0
  CONFIG_LIBC_GAISTRERROR_ERRNUM=y
  CONFIG_LIBC_FTOK_VFS_PATH="/var/ftok"
  CONFIG_LIBC_MEMFD_ERROR=y
  CONFIG_LIBC_TEMPBUFFER=y
  CONFIG_LIBC_MAX_TEMPBUFFER=2
  CONFIG_LIBC_TEMPBUFFER_MALLOC=y
  CONFIG_LIBC_MUTEX_BACKTRACE=0
  CONFIG_BUILTIN=y
  CONFIG_STREAM_OUT_BUFFER_SIZE=64
  CONFIG_STREAM_HEXDUMP_BUFFER_SIZE=128
  CONFIG_STREAM_BASE64_BUFFER_SIZE=128
  CONFIG_LIBC_INLINE_QUEUE=y
  CONFIG_LIBM_TOOLCHAIN=y
  CONFIG_HAVE_CXX=y
  CONFIG_HAVE_CXXINITIALIZE=y
  CONFIG_LIBCXXNONE=y
  CONFIG_LIBMINIABI=y
  CONFIG_CXX_STANDARD="gnu++17"
  CONFIG_CXX_NO_LOCALIZATION=y
  CONFIG_BUILTIN_TOOLCHAIN=y
  CONFIG_COVERAGE_NONE=y
  CONFIG_COVERAGE_DEFAULT_PREFIX_STRIP="0"
  CONFIG_COVERAGE_DEFAULT_PREFIX="/data"
  CONFIG_PROFILE_NONE=y
  CONFIG_CANUTILS_CANLIB=y
  CONFIG_CANUTILS_LELYCANOPEN=y
  CONFIG_CANUTILS_LELYCANOPEN_URL="https://gitlab.com/lely_industries/lely-core/-/archive/master/"
  CONFIG_CANUTILS_LELYCANOPEN_VERSION="637de61625c9e8af2b9598109d6fb0e3a2829ce6"
  CONFIG_CANUTILS_LELYCANOPEN_NMTCFG=y
  CONFIG_CANUTILS_LELYCANOPEN_IOLIB=y
  CONFIG_NETUTILS_NETINIT=y
  CONFIG_NETINIT_IPADDR=0x0a000002
  CONFIG_NETINIT_DRIPADDR=0x0a000001
  CONFIG_NETINIT_NETMASK=0xffffff00
  CONFIG_NETUTILS_NETLIB=y
  CONFIG_NSH_LIBRARY=y
  CONFIG_NSH_PROMPT_STRING="nsh> "
  CONFIG_NSH_PROMPT_MAX=32
  CONFIG_NSH_PROMPT_ENV="PS1"
  CONFIG_NSH_PROMPT_SUFFIX="> "
  CONFIG_NSH_READLINE=y
  CONFIG_NSH_MAXARGUMENTS=7
  CONFIG_NSH_NESTDEPTH=3
  CONFIG_NSH_ALIAS=y
  CONFIG_NSH_ALIAS_MAX_AMOUNT=1
  CONFIG_NSH_PIPELINE=y
  CONFIG_NSH_BUILTIN_APPS=y
  CONFIG_NSH_DISABLE_DATE=y
  CONFIG_NSH_DISABLE_LOSMART=y
  CONFIG_NSH_DISABLE_LOMTD=y
  CONFIG_NSH_DISABLE_MB=y
  CONFIG_NSH_DISABLE_MH=y
  CONFIG_NSH_DISABLE_MW=y
  CONFIG_NSH_DISABLE_TIMEDATECTL=y
  CONFIG_NSH_CODECS_BUFSIZE=128
  CONFIG_NSH_PROC_MOUNTPOINT="/proc"
  CONFIG_NSH_FILEIOSIZE=512
  CONFIG_NSH_SYSINITSCRIPT="init.d/rc.sysinit"
  CONFIG_NSH_INITSCRIPT="init.d/rcS"
  CONFIG_NSH_SCRIPT_REDIRECT_PATH=""
  CONFIG_NSH_CONSOLE=y
  CONFIG_NSH_ARCHINIT=y
  CONFIG_NSH_NETINIT=y
  CONFIG_NSH_WGET_BUFF_SIZE=512
  CONFIG_SYSTEM_DD=y
  CONFIG_SYSTEM_DD_PROGNAME="dd"
  CONFIG_SYSTEM_DD_PRIORITY=100
  CONFIG_SYSTEM_DD_STACKSIZE=2048
  CONFIG_SYSTEM_DD_STATS=y
  CONFIG_SYSTEM_NSH=y
  CONFIG_SYSTEM_NSH_PRIORITY=100
  CONFIG_SYSTEM_NSH_STACKSIZE=2048
  CONFIG_SYSTEM_NSH_PROGNAME="nsh"
  CONFIG_SYSTEM_READLINE=y
  CONFIG_READLINE_HAVE_EXTMATCH=y
  CONFIG_READLINE_ECHO=y

Host system OS:
  Linux ryzen 6.12.93 #1-NixOS SMP PREEMPT_DYNAMIC Tue Jun  9 10:26:06 UTC 2026 x86_64

Host system PATH:
  /nix/store/p2zq0yfly2i49aagmzznnqhnzpf801jy-curl-8.20.0-dev/bin
  /nix/store/c22rvsjx1qdlh4rq1zqcs5bgazg47pc9-brotli-1.1.0/bin
  /nix/store/hr3spb8jqbl518r07zpzf5mpq1xvmipv-krb5-1.22.1-dev/bin
  /nix/store/n4pzsddppf7sh8r7vx7qfrh41v2nmady-krb5-1.22.1/bin
  /nix/store/96ndqjyycxs026fqx27cwbvjjlnm4z20-nghttp2-1.68.1/bin
  /nix/store/g48l0gglycfcfxzknz5a9rhsn05sdb82-libidn2-2.3.8-bin/bin
  /nix/store/3ffi89rlwcqww9x71pf95miyik5jhnv9-openssl-3.6.2-bin/bin
  /nix/store/asjpw9mi2j10amdqb0x4p83h7a5nvyyr-libpsl-0.21.5/bin
  /nix/store/qap9s0ndw3bc2x54r44wbvr7m1kz69k5-zstd-1.5.7-bin/bin
  /nix/store/4sq0awhi6w9zw9qb08amnams96a0kwp5-zstd-1.5.7/bin
  /nix/store/2nmfyy3jjgn2m8d1p7lwbbxakl4ciwfc-curl-8.20.0-bin/bin
  /nix/store/5mrhn7sj5hmx1sqqj1568sya450yarvx-patchelf-0.15.2/bin
  /nix/store/8v97ngkcpfzgghwnnr7fsz33p2x22gy9-gcc-wrapper-14.3.0/bin
  /nix/store/ipr6y28viyqkhkg58rdvy27m01q5j5nh-gcc-14.3.0/bin
  /nix/store/6s5jp0cchd4jjq474xkfymj18l6i5x95-glibc-2.40-224-bin/bin
  /nix/store/hqkszxk2c0cxvd04xa4gsaqs182dw8l2-coreutils-9.8/bin
  /nix/store/iz5lckcsg66r223si2gck7csk2hihj0m-binutils-wrapper-2.44/bin
  /nix/store/i7mdvmliqcb5lz0nqija8rq55vws6gi8-binutils-2.44/bin
  /nix/store/i5zf2arkflazjnxv2y46fmhyfwzl5hdy-cmake-4.1.2/bin
  /nix/store/xlyizfr1j0c5v3yzv73zds9b0jzkqv8h-ninja-1.13.1/bin
  /nix/store/fnvsac4yaw2146ig4p54xnnm6b6alkjw-gnumake-4.4.1/bin
  /nix/store/hxym6vx51l4jfgj2379x1aphdxs94zf6-clang-tools-21.1.7/bin
  /nix/store/j1p5iwbcry6gbzr747972zb2v4j5vq1f-gcc-arm-embedded-14.3.rel1/bin
  /nix/store/dbqq2z49s4d8fqvs3bia1qzvmxv23d8z-automake-1.18.1/bin
  /nix/store/qjcy2za2wd0sgvhipmjka6dvhvbwvmaz-bison-3.8.2/bin
  /nix/store/60j9yqfi7gzf6g1kg0gz2z050bljc50h-gnum4-1.4.20/bin
  /nix/store/mkrwg4hn1ckr1baqd70zvmicx04y31x9-flex-2.6.4/bin
  /nix/store/ygwzfgf2g8dyxhjvjmk3j7c8j4bsdb89-genromfs-0.5.2/bin
  /nix/store/7cfsh0m5d6fwkdd6ry3d3aknmbmjykcs-gettext-0.25.1/bin
  /nix/store/s83x515zdi9dl95a8jgg4nyhhy62dvs9-gperf-3.3/bin
  /nix/store/pyqb42g207b8aad5jgdzcca2ckrai4yn-expat-2.8.1-dev/bin
  /nix/store/s7hmpxm2pm4zxsbm0x1fhlzrdvr2v4ga-ncurses-6.5-dev/bin
  /nix/store/ffyzkisqs4vc4mg28bwwlyqjf8i9ph6b-ncurses-6.5/bin
  /nix/store/bsp2f26x73r2nwkcbakvsvazm6y5sdhq-kconfig-frontends-4.11.0.1/bin
  /nix/store/fkiffwbcr41i8k0w33pd6hxz3p5ff5ib-python3.13-kconfiglib-14.1.0/bin
  /nix/store/vm6nxpp97fxgczw00bwkxdqdm2an3n95-python3-3.13.12/bin
  /nix/store/39gydzj63ckwxh1ccdwagdmgzqcq7zib-python3-3.13.12-env/bin
  /nix/store/mgh163i8h0c6xh46wa5hw0rxn62bqvml-git-2.51.2/bin
  /nix/store/fa0afx6zkpn15f77r3d2afcn3xv3m2f8-wget-1.25.0/bin
  /nix/store/qhr81gkafwyczvwk173ymahhwrilfv5i-pkg-config-wrapper-0.29.2/bin
  /nix/store/lxfap3kkb026d6z0pydyn4b08zfxlvad-ncurses-abi5-compat-6.5-dev/bin
  /nix/store/dfdfw2cx3l56x2ihgv4314xnaxz13p9c-ncurses-abi5-compat-6.5/bin
  /nix/store/a6yca4wqrjrdjg2wflpya4vly5w86fxl-dfu-util-0.11/bin
  /nix/store/kjf6cnkbv60fwfp3mry37igbfqsa49hg-esp32ulp-elf-esp-idf-v5.5.2/bin
  /nix/store/wf4vfk133bn6mbszv54p3n88545pp1qg-openocd-esp32-esp-idf-v5.5.2/bin
  /nix/store/n5yzwwxrh956zpiv7xk1vp0xsgp3vy1m-xtensa-esp-elf-esp-idf-v5.5.2/bin
  /nix/store/wwsgyyskjfj21x3g1z3gmmfiz2mslifz-xtensa-esp-elf-gdb-esp-idf-v5.5.2/bin
  /nix/store/jslbsz86s67ffz1brph7m8m6943sn70n-tio-3.9/bin
  /nix/store/9hhbx3l9x3d5lqz50qwbxb65szfl3lwz-esptool-5.1.0/bin
  /nix/store/szk2hjk1gq582yfzmdkcvq9fqs1i5qcg-python3.13-intelhex-2.3.0/bin
  /nix/store/il4sa3kiidg3bdq3jln6687070zm3v7q-python3.13-pyserial-3.5/bin
  /nix/store/wmxxklybwiipq6bnwidxmgiyqk28nflz-python3.13-rich-click-1.8.9/bin
  /nix/store/ps4m2q8sab1262b7q79y48wcm2bs1c75-python3.13-markdown-it-py-3.0.0/bin
  /nix/store/qp77zp0jkshv9pmcpf53g5q46sy8wa56-python3.13-pygments-2.19.2/bin
  /nix/store/w13gvgp7k0ax7pnjr11csrwg0rcbykg1-espflash-4.2.0/bin
  /nix/store/hqkszxk2c0cxvd04xa4gsaqs182dw8l2-coreutils-9.8/bin
  /nix/store/8xhaz2ysixijy8sgzmwmhlialqqind1p-findutils-4.10.0/bin
  /nix/store/cw3a3yz8vrfsz53ypjw8hspy5hzzx2py-diffutils-3.12/bin
  /nix/store/4zi0y6cmpjnbhmgrx7vfgqkyv5z1z4z0-gnused-4.9/bin
  /nix/store/k3wiv3qqa4y0im5v1iq2jy4h9cm32dfc-gnugrep-3.12/bin
  /nix/store/88d79vz87a5g3wsclabrgqd93jqh5yv8-gawk-5.3.2/bin
  /nix/store/xl6pd2spjajwcpq09fnmnjv5psqmlmma-gnutar-1.35/bin
  /nix/store/rhp8ph1q699irvmacp27x3sw84slk32i-gzip-1.14/bin
  /nix/store/g36v1ss2ywkqp751wlkhnd8n9zaqkg2w-bzip2-1.0.8-bin/bin
  /nix/store/djjc508wnfdwmkv5xrxxqx5r2v09yxfp-gnumake-4.4.1/bin
  /nix/store/3hgg7pr65imdrifqqh3flg3arvkc2r22-bash-5.3p3/bin
  /nix/store/wgnxx41q3d2cx6j5wk2s5bkjyzrr3ggn-patch-2.8/bin
  /nix/store/3x3p3mhwn18q7qplxn34i13rdmswi0qm-xz-5.8.3-bin/bin
  /nix/store/018l43lbsqbrsasrfdcxxb5s4s161fkr-file-5.45/bin
  /nix/store/k017inlry2f8qvbybmf26k6jsayyrq9s-esp-idf-v5.5.2/tools
  /nix/store/k017inlry2f8qvbybmf26k6jsayyrq9s-esp-idf-v5.5.2/components/espcoredump
  /nix/store/k017inlry2f8qvbybmf26k6jsayyrq9s-esp-idf-v5.5.2/components/partition_table
  /nix/store/k017inlry2f8qvbybmf26k6jsayyrq9s-esp-idf-v5.5.2/components/app_update
  /run/wrappers/bin
  /nix/profile/bin
  /etc/profiles/per-user/roman/bin
  /nix/var/nix/profiles/default/bin
  /run/current-system/sw/bin
  /nix/store/iz5lckcsg66r223si2gck7csk2hihj0m-binutils-wrapper-2.44/bin
  /nix/store/x065ywa43gszk7i3iw1i1klbayzh636c-hyprland-qtutils-0.1.5/bin
  /nix/store/a0dpq6c9flsi5lmszckwxphda3il64kg-pciutils-3.14.0/bin
  /nix/store/kp7sggj64lydb246p182gv4whq2dbxis-pkgconf-wrapper-2.4.3/bin
  /nix/store/1rgzy3gya1p2j1z2qr2h5kc2whrhchwa-ghostty-1.3.1/bin

Espressif specific information:

Toolchain version:
  esp32: Bootloader image not found
  esp32s2: Bootloader image not found
  esp32s3: Bootloader image not found
  esp32c2: Bootloader image not found
  esp32c3: Bootloader image not found
  esp32c6: Bootloader image not found
  esp32h2: Bootloader image not found

Toolchain version:
  clang: Not found
  gcc: gcc (GCC) 14.3.0
  xtensa-esp32-elf-gcc: xtensa-esp-elf-gcc (crosstool-NG esp-14.2.0_20251107) 14.2.0
  xtensa-esp32s2-elf-gcc: xtensa-esp-elf-gcc (crosstool-NG esp-14.2.0_20251107) 14.2.0
  xtensa-esp32s3-elf-gcc: xtensa-esp-elf-gcc (crosstool-NG esp-14.2.0_20251107) 14.2.0
  riscv-none-elf-gcc: Not found
  riscv32-esp-elf-gcc: Not found
  riscv64-unknown-elf-gcc: Not found

Esptool version:
  4.9.0

HAL version:
  sync/master.a-nuttx-20250610-20-gb9472bd56b9

CHIP ID:
  ESP32 has no Chip ID. Reading MAC instead.

Flash ID:
  Manufacturer: 68
  Device: 4016

Security information:
  Not found

Flash status:
  0x0000

MAC address:

```

### Verification

- [x] I have verified before submitting the report.

--- a/canutils/lely-canopen/Kconfig
+++ b/canutils/lely-canopen/Kconfig
@@ -51,6 +51,7 @@ config CANUTILS_LELYCANOPEN_OBJUPLOAD
 config CANUTILS_LELYCANOPEN_SDEV
 	bool "Lely CANopen enable static device description support"
 	default n
+	depends on CANUTILS_LELYCANOPEN_OBJNAME
 
 config CANUTILS_LELYCANOPEN_CSDO
 	bool "Lely CANopen enable Client-SDO support"
@@ -67,6 +68,7 @@ config CANUTILS_LELYCANOPEN_TPDO
 config CANUTILS_LELYCANOPEN_MPDO
 	bool "Lely CANopen enable Multiplex PDO support"
 	default n
+	depends on CANUTILS_LELYCANOPEN_TPDO
 
 config CANUTILS_LELYCANOPEN_SYNC
 	bool "Lely CANopen enable SYNC support"
@@ -95,14 +97,17 @@ config CANUTILS_LELYCANOPEN_MASTER
 config CANUTILS_LELYCANOPEN_NG
 	bool "Lely CANopen enable node guardian support"
 	default n
+	depends on CANUTILS_LELYCANOPEN_EMCY
 
 config CANUTILS_LELYCANOPEN_NMTBOOT
 	bool "Lely CANopen enable NMT boot slave support"
 	default n
+	depends on CANUTILS_LELYCANOPEN_MASTER && CANUTILS_LELYCANOPEN_CSDO
 
 config CANUTILS_LELYCANOPEN_NMTCFG
 	bool "Lely CANopen enable NMT configuration request support"
 	default n
+	depends on CANUTILS_LELYCANOPEN_MASTER && CANUTILS_LELYCANOPEN_CSDO
 
 config CANUTILS_LELYCANOPEN_GW
 	bool "Lely CANopen enable gateway support"

